spec/index.html

```diff
@@ -4741,7 +4741,9 @@ <h3>Filter Evaluation</h3>
             "xsd:boolean (EBV)" in the operator mapping table below), are coerced to
             <code>xsd:boolean</code> using the <a href="#ebv">EBV rules</a> in section 17.2.2.
           </li>
-          <li>Apart from <a href="#func-bound">BOUND</a>, <a href="#func-coalesce">COALESCE</a>,
+          <li>Apart from the <a href="#func-forms">functional forms</a>
+            <a href="#func-bound">BOUND</a>, <a href="#func-coalesce">COALESCE</a>,
+            <a href="#func-if">IF</a>, <a href="#func-in">IN</a>, <a href="#func-not-in">NOT IN</a>, 
```

Comment:
It says in the rest of the text of this point, "and operators" so that would sweep up `||` and `&&`. 

```suggestion
            <a href="#func-if">IF</a>, <a href="#func-in">IN</a>, <a href="#func-not-in">NOT IN</a>, 
            <a href="#func-logical-or">logical-or</a> (<code>||</code>),
            <a href="#func-logical-and">logical-and</a> (<code>&amp;&</code>),
```

So strictly, either include in these in the list or drop the "operators".

A possibility is split into two point to avoid one large one - one for function-like appearance and operator-lie appearance.